apps/sms/js/thread_list_ui.js

```diff
@@ -282,6 +286,71 @@ var ThreadListUI = {
         }
 
         break;
+      case 'contextmenu':
+        if (this.inEditMode) {
+          return;
+        }
+
+        parent = event.target.parentNode;
+        parentThreadId = parent.dataset.threadId;
+        var thread = Threads.get(parentThreadId);
+
+        if (parentThreadId) {
+          event.preventDefault();
+          event.stopPropagation();
+          // Show options per single thread
+          var params = {
+            type: 'action',
+            header: { l10nId: 'thread-options' },
+            items:[]
+          };
+          threadIdToDelete.push(parentThreadId);
+
+          params.items.push(
+            {
+              l10nId: 'delete-thread',
+              method: function deleteThread(threadIdToDelete) {
+                ThreadListUI.performDeletion(threadIdToDelete);
```

Comment:
or even: `method: this.performDeletion.bind(this, threadIdToDelete)` and remove the `params` property below (that I'd like to kill because it's useless)